docs/kwc-cms/customize-components/child-components.md

```diff
@@ -24,4 +24,14 @@ To do that you have to modify the generators setting of the component:
     $ret['generators']['child']['component'] = 'Kwc_Basic_Text_Component';
     
   
-Check the parent component class to see which is the correct way.
\ No newline at end of file
+Check the parent component class to see which is the correct way.
+
+## Child Settings
+You can change the settings of a child component without creating a new class for it by using `childSettings`.
```

Comment:
pls describe that for example text stands for the child component key

and when you pass '_merge' => 'reset' the parent settings will be resetted

Reply:
@kaufmo I added those changes in 99a9bc2a1b72c365ebe554cff69200cf467d8e6d.